### ISSUE TITLE
.NET 5 and C#9 version

### ISSUE THREAD

**Issue by ivanpaulovich** [ivanpaulovich/clean-architecture-manga#184]
Hi all,

I am excited to inform you that I started today the [.NET 5 and C#9](https://github.com/ivanpaulovich/clean-architecture-manga/tree/dotnet5) version of the Clean Architecture Manga.

This blog post [Welcome to C# 9](https://devblogs.microsoft.com/dotnet/welcome-to-c-9-0/) shows interesting C#9 features. For instance, I am looking forward to using [Covariant Returns](https://devblogs.microsoft.com/dotnet/welcome-to-c-9-0/#covariant-returns) on repositories and presenters by removing the typecasting.

This Issue is also an open invitation for the community to contribute and learn together. Have nice weekend folks!

**Comment by fdolsky321** [comment]
Hi @ivanpaulovich any reason of removing dotnet5 branch? :) Btw. thanks a lot for all your effort and keep going :).

**Comment by ivanpaulovich** [comment]
Hey @fdolsky321 :)

I felt that the previous branch was too outdated so I created a new branch for .NET 5, I quickly migrated the Accounts API  https://github.com/ivanpaulovich/clean-architecture-manga/commit/7c98284a428d854392d27cc496e23a577ed3e725 and I already set it to the default branch.

The previous master branch was renamed to [dotnet3.1](https://github.com/ivanpaulovich/clean-architecture-manga/tree/dotnet3.1).

The funny thing was that I recorded a video doing so [.NET 5 Migration with VSCode and Rider](https://www.youtube.com/watch?v=gWWnKrBp6EU).

Of course the C# 9 features will come slowly.
